src/function-builder-detectors/netlify-lambda.js

```diff
@@ -17,7 +25,9 @@ module.exports = function() {
 
   for (const key in scripts) {
     const script = scripts[key]
-    const match = script.match(/netlify-lambda build (\S+)/)
+
+    const match = script.match(/netlify-lambda build.* (\S+)\s*$/)
```

Comment:
Could we switch to something like https://github.com/substack/node-shell-quote#parse?